docs/reference/commandline/service_create.md

```diff
@@ -363,16 +363,34 @@ volumes in a service:
     <td></td>
     <td>
       <p>The Engine mounts binds and volumes <tt>read-write</tt> unless <tt>readonly</tt> option
-      is given when mounting the bind or volume.
+      is given when mounting the bind or volume. Note that setting <tt>readonly</tt> for a
+      bind-mount does not make its submounts <tt>readonly</tt> on the current Linux implementation. See also <tt>bind-nonrecursive</tt>.
       <ul>
         <li><tt>true</tt> or <tt>1</tt> or no value: Mounts the bind or volume read-only.</li>
         <li><tt>false</tt> or <tt>0</tt>: Mounts the bind or volume read-write.</li>
       </ul></p>
     </td>
   </tr>
+</table>
+
+#### Options for Bind Mounts
+
+The following options can only be used for bind mounts (`type=bind`):
+
+
+<table>
+  <tr>
+    <th>Option</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><b>bind-propagation</b></td>
+    <td>
+      <p>See [Bind Propagation section](#bind-propagation).</p>
```

Comment:
Looks like this needs a HTML link, not Markdown 😕 

```suggestion
      <p>See the <a href="#bind-propagation">bind propagation section</a>.</p>
```

Currently renders as 
<img width="877" alt="screen shot 2019-01-09 at 16 27 08" src="https://user-images.githubusercontent.com/1804568/50909181-9eb01580-142b-11e9-9412-0d9cec643be8.png">